input/resources/au-core-procedure.xml

```diff
@@ -67,7 +59,6 @@
         <targetProfile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient"/>
        <targetProfile value="http://hl7.org/fhir/StructureDefinition/RelatedPerson"/>
       </type>
-      <mustSupport value="true"/>
     </element>
     <element id="Procedure.asserter">
```

Comment:
Remove Procedure.asserter entirely from the differential in the input file. At this time we do not set every referenced resource to be an AU Core profile where one exists.

Reply:
Done. Removed Procedure.asserter entirely from the differential in the input file.